### PR TITLE
Trigger classify on onboarding k8s system scan

### PR DESCRIPTION
**Pull request by allisonking**
Closes https://github.com/ethyca/fidesctl-plus/issues/190

### Code Changes

* [x] Moves storing the way the user is adding systems (aws, runtime scanner, okta, etc.) into redux
* [x] Adds a `classify=true` query param to the existing scan endpoint (backend work still [in progress](https://github.com/ethyca/fidesctl-plus/pull/304))
* [x] Refactors the ScanResults page to `RegisterSystems`. This was more useful when I thought `/classify-systems` wasn't going to be its own page and was just going to be a view stored in state. That's no longer the case after talking with design today, but I think the refactor is still a bit cleaner
* [x] Adds a new page `/classify-systems` with a classification table
* [x] Refactors the dataset classification badge into its own component so we can reuse it in this systems classification table
* [x] Renders the classification status in the table 
* [x] Adds cypress tests

### Steps to Confirm

* [ ] The backend endpoints don't quite exist yet, so your best bet is to run the cypress test "Allows calling the runtie scanner with classify". Note that the fixture only has two classification instances, hence why there are only two rows with "Awaiting Review" and everything else is "Unknown"

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation Updated:
  * [ ] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  * [ ] documentation issue created (tag docs-team to complete issue separately)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [x] Update `CHANGELOG.md`

### Description Of Changes

https://user-images.githubusercontent.com/24641006/201762900-7f867111-870b-4dac-bab6-0c36b521cfb1.mov

### CODE REVIEW

File: clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts [hunk 1]
```diff
@@ -71,11 +78,14 @@ describe.skip("Config wizard with plus settings", () => {
         const { body } = interception.request;
         expect(body.length).to.eql(numSystems);
       });
-      /* This will redirect to localhost:3000/datamap instead of localhost:4000/datamap
-       * which is not actually what we want.
-       * However, it is difficult to test across zones, so all we can do is assert the path is right
-       */
-      cy.url().should("contain", "datamap");
+      cy.getByTestId("systems-classify-table");
```
Comment (ssangervasi) [inline clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts:81]: I tried running the tests by removing the `describe.skip` and enabling the feature flag to make the page appear. The test fails here, looking like this:
Screen Shot 2022-11-11 at 15 17 11

File: clients/admin-ui/src/features/config-wizard/AuthenticateRuntimeForm.tsx [hunk 2]
```diff
@@ -19,7 +19,9 @@ import ScannerLoading from "./ScannerLoading";
 const AuthenticateRuntimeForm = () => {
   const dispatch = useAppDispatch();
   const toast = useToast();
-  const { data, error: queryError } = useGetScanResultsQuery();
+  const { data, error: queryError } = useGetScanResultsQuery({
```
Comment (ssangervasi) [inline clients/admin-ui/src/features/config-wizard/AuthenticateRuntimeForm.tsx:22]: Interesting that this is a GET request. And by interesting I mean "yikes". It kicks off a process that will, among other things, create ClassifyInstances, right? Unless I'm missing something.

Not a UI code problem, if that's case, just raised my eyebrows. Maybe the component shouldn't be called "Authenticate", though, if were really kicking it off just by entering this step.

Comment (allisonking) [reply clients/admin-ui/src/features/config-wizard/AuthenticateRuntimeForm.tsx:22]: ahh right, that's a good point. we had planned to change it over to PUT as part of https://github.com/ethyca/fidesctl-plus/issues/195 (which will write scan history), but you're right that we should do it now that we're writing classify instances. @SteveDMurphy 's being a hero and said he'll change it to a PUT as part of https://github.com/ethyca/fidesctl-plus/pull/304, and then I'll update the frontend in https://github.com/ethyca/fides/pull/1752

Yeah, it probably shouldn't be called Authenticate. it was the easiest way to fit it into the existing onboarding flow though. I think https://github.com/ethyca/fides/issues/1514 will make the "steps" more true to each flow, and then this component won't have to be stuffed into "Authenticate". I'll rename it for now though 👍

File: clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts (hunk 1 above)
Comment (ssangervasi) [reply clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts:97]: I'm still getting an error just on `Allows calling the runtime scanner with classify`

**Review by ssangervasi — APPROVED** [review]
Just one funky test

**Comment by allisonking** [comment]
> Just one funky test

confirmed with @ssangervasi that it works after https://github.com/ethyca/fides/pull/1720/commits/2bf3d53f110992c9201723ee839803a8fd531a55, so merged! thanks for the thorough review 🙏

**Pull request merged**